src/constants.ts

```diff
@@ -126,9 +126,9 @@ const Constants = {
         integrationDelayTimeout: 5000, // Milliseconds for forcing the integration delay to un-suspend event queueing due to integration partner errors
         maxCookieSize: 3000, // Number of bytes for cookie size to not exceed
         aliasMaxWindow: 90, // Max age of Alias request startTime, in days
-        uploadInterval: 0, // Maximum milliseconds in between batch uploads, below 500 will mean immediate upload
+        uploadInterval: '0', // Maximum milliseconds in between batch uploads, below 500 will mean immediate upload.  This is a string because that is how the server returns it
```

Comment:
As per our conversation on Zoom, this should be a number. All types used within the SDK should conform to JS, even if the server returns only strings.